sympy/physics/control/lti.py

```diff
@@ -1075,6 +1075,46 @@ def eval_frequency(self, other):
         argnew = TransferFunction(arg_num, arg_den, self.var).to_expr()
         return argnew.expand()
 
+    def bandwidth(self, power_gain=0.5):
+        """
+        Returns the system bandwidth for a given power gain.
+        If power_gain is not provided, it returns the 3 dB Bandwidth or
+        more precisely called 3.01 dB Bandwidth, which means power gain is 0.5.
+
+        Examples
+        ========
+
+        >>> from sympy import Symbol
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> s = Symbol('s')
+        >>> tf1 = TransferFunction(1, s + 1, s)
+        >>> tf1.bandwidth()
+        1.00000000000000
+
+        We can also use this method with symbolic TransferFunction.
+
+        >>> a = Symbol('a', real=True, positive=True)
+        >>> tf2 = TransferFunction(1, s - a, s)
+        >>> tf2.bandwidth()
+        1.0*a
+
+        Notes
+        =====
+
+        When using symbolic TransferFunction, make sure to define symbols along with
+        their signs properly.
+        """
```

Comment:
Please cite a reference for this "bandwidth" calculation. I'm not immediately familiar with this and what the "power gain" is.

Also, this should be in a separate PR, as it is not related to PID.